lib/datasource/galaxy-collection/index.spec.ts

```diff
@@ -0,0 +1,148 @@
+import fs from 'fs';
+import { getPkgReleases } from '..';
+import * as httpMock from '../../../test/http-mock';
+
+import { id as datasource } from '.';
+
+const communityKubernetesBase = fs.readFileSync(
+  'lib/datasource/galaxy-collection/__fixtures__/community_kubernetes_base.json',
+  'utf8'
+);
+const communityKubernetesVersions = fs.readFileSync(
+  'lib/datasource/galaxy-collection/__fixtures__/community_kubernetes_versions.json',
+  'utf8'
+);
+const communityKubernetesDetails121 = fs.readFileSync(
+  'lib/datasource/galaxy-collection/__fixtures__/community_kubernetes_version_details_1.2.1.json',
+  'utf8'
+);
+const communityKubernetesDetails120 = fs.readFileSync(
+  'lib/datasource/galaxy-collection/__fixtures__/community_kubernetes_version_details_1.2.0.json',
+  'utf8'
+);
+const communityKubernetesDetails0111 = fs.readFileSync(
+  'lib/datasource/galaxy-collection/__fixtures__/community_kubernetes_version_details_0.11.1.json',
+  'utf8'
+);
+
+const baseUrl = 'https://galaxy.ansible.com:443/';
+
+describe('datasource/galaxy-collection', () => {
```

Comment:
```suggestion
describe(getName(__filename), () => {
```

Reply:
Is this a new standard? I ask because this breaks my debugger during unit tests. I use Webstorm btw.

Reply:
yes, what's wrong with this? it works fine on cli and vscode.

Reply:
The debugger missed all breaking points. 

Seems to be a Nodejs issue. After reinstalling Nodejs it started working again.